decidim-elections/app/views/decidim/elections/votes/new.html.erb

```diff
@@ -4,6 +4,7 @@
   data-authority-public-key="<%= authority_public_key %>"
   data-election-unique-id="<%= election_unique_id %>"
   data-voter-id="<%= voter_id %>"
+  data-ballot-style-id="<%= ballot_style_id || "null" %>"
```

Comment:
I'm not very comfortable with that `"null"` haha. I assume that when you use `.data("ballotStyleId")` it gives you a JS `null` but stil... let me give you another alternative:

```
data-voter-id="<%= voter_id %>"
<%= ballot_style_id.present? ? "data-ballot-style-id=\"#{ballot_style_id}\"" : nil %>
```

It looks a bit uglier but for me, is more explicit: when `ballot_style_id` is not present you don't add that data attribute. Feel free to use it or not 😄